src/contributors/joaolfelicio.json

```
@@ -0,0 +1,6 @@
+{
+    "name": "Joao Felicio",
+    "desc": "Software Engineer in love with programming.",
+    "github" : "https://github.com/joaolfelicio",
```

```suggestion
    "github": "https://github.com/joaolfelicio",
```
Remove unwanted `white-space`.